docs/source/markdown/podman-create.1.md

```diff
@@ -22,6 +22,28 @@ Default settings for flags are defined in `containers.conf`. Most settings for
 remote connections use the server's containers.conf, except when documented in
 man pages.
 
+## IMAGE
+
+ The `image` uses a "transport":"details" format.
```

Comment:
"The image is specified using `transport:path` format. If no transport is specified, the `docker` transport will be used by default. For remote Podman, `docker` is the only allowed transport."

Reply:
Fixed.